com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.MixedReality.Toolkit.Input.Simulation;
 using Microsoft.MixedReality.Toolkit.Subsystems;
-using System;
+using System.Collections.Generic;
```

Comment:
Is this still needed with the latest changes?

Reply:
Fixed